Rover/JetsonGStreamerServer/gserver.cpp

```diff
@@ -0,0 +1,227 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "gserver.h"
+#include <unordered_map>
+#include <thread>
+#include <vector>
+#include <signal.h> 
+
+extern "C" 
+{
+  #include <gst/gst.h>
+  #include <gst/gstprotection.h>
+}
+
+GMainLoop * loop;
```

Comment:
Standardize pointer spacing. Either GMainLoop* loop, GMainLoop * loop, or GMainLoop *loop. Immediately below you do it the third way, here you do it the second way. I prefer the third way, but it doesn't matter as long as you're consistent.